kubeflow/rv/rv_kubeflow/spacenet_vegas.py

```diff
@@ -0,0 +1,230 @@
+# flake8: noqa
+
```

Comment:
I had to add this modified copy of the Spacenet Vegas example because I needed to change the hyperparameters so it would run even faster in test mode.